samples/gazell/gzll_ack_payload/README.rst

```diff
@@ -7,11 +7,25 @@ Gazell ACK Payload
    :local:
    :depth: 2
 
+This sample shows basic Gazell communication and demonstrates how to send payloads and acknowledgments.
```

Comment:
When reading further down to Building and running, I noticed this:

```
The Device sample is under :file:`samples/gazell/gzll_ack_payload/device` in the |NCS| folder structure.
The Host sample is under :file:`samples/gazell/gzll_ack_payload/host` in the |NCS| folder structure.
```

So does this mean there are actually two samples? We can't have two samples covered under one sample...
This is also the case for the other Gazell sample, which makes me think that the actual samples are Gazell Host and Gazell Device, which can work in two different scenarios (ACK payload and Gazell Dynamic Pairing). Unless there are effectively 4 different samples (Host for ACK payload, Device for ACK payload, Host for Gazell Dynamic Pairing, Device for Gazell Dynamic Pairing), which would require documenting all 4 of them on the same level, and splitting them in the code repository.

I suggest you check with the Gazell developers about reworking the code, because we cannot allow samples made of multiple other samples. There was a similar problem with the networking samples, such as Zigbee or BT mesh, but this was solved by documenting each of them separately, and then adding information that they can form a network together with other samples. See for example here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/zigbee/network_coordinator/README.html

Reply:
Has been split into 4 different samples. (Divided the two existing ones into Host and Device).

Reply:
Sorry for interrupt. The Gazell samples README.rst follows the ESB sample layout. The ESB sample takes 2 DKs as a transmitter and a receiver. Yes, it differs from other samples. Anyway, feel free to modify it.

https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/samples/esb/README.html

Reply:
That is a good point @leewkb4567. This was also brought to my attention by @grochu and I have created a Jira ticket for it, and there may be other sample pages that are in a similar situation that need to be looked too. I will discuss with others, including my fellow technical writers to see what they think as well.